src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -1,3 +1,4 @@
 REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;
+-- touch
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO "isyfo-analyse";
```

Comment:
For å trigge den på nytt?

Reply:
Ja, vi får ikke gjort select på tabellene før dette har kjørt på nytt.